Dockerfile

```diff
@@ -506,6 +506,8 @@ ADD patches/imagemagick-policy.xml /etc/ImageMagick-6/policy.xml
 RUN jupyter-nbextension disable nb_conda --py --sys-prefix && \
     jupyter-serverextension disable nb_conda --py --sys-prefix && \
     python -m nb_conda_kernels.install --disable
+    
+RUN jupyter-serverextension disable caip_notebooks_serverextension
```

Comment:
1. I'm not very familiar with CAIP integration, so I prefer Vincent to confirm this change.
2. Could you add a comment, maybe a link to a bug?

Reply:
There's a link to the bug in the PR, but I'll add one in the code as well.

The CAIP integration is installed in the base image, and it's a jupyter extension that seems to only work when the notebook is hosted on caip.

Reply:
We kept it for when people "upgrade" their notebook to run on caip. However, if it adds 6s, we should remove it.

Does this extension tell you information beyond what GCE machine your CAIP notebook run on?